examples/tutorials/colabs/Habitat_Lab.ipynb

```diff
@@ -595,4 +595,4 @@
  },
  "nbformat": 4,
  "nbformat_minor": 4
-}
+}
```

Comment:
Is that change needed? I think pre-commit should complain about this.

Reply:
Oh, no
This is some how automatically done by my IDE. I'll fix.

Reply:
done